docs/install/command-line.md

```diff
@@ -89,43 +89,69 @@ The IPFS team manages the [dist.ipfs.tech website](https://dist.ipfs.tech/) to h
 
 ### macOS
 
-:::tip M1-based Macs
-You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead of the `amd64` binary listed in these instructions.
-:::
+1. Decide which macOS binary to download from [`dist.ipfs.tech`](https://dist.ipfs.tech/#kubo) by determining whether your system uses Apple or Intel CPUs. On most MacOS systems, you can determine the system specs by doing the following:
 
-1. Download the macOS binary from [`dist.ipfs.tech`](https://dist.ipfs.tech/#kubo).
+   a. In the upper left hand corner of your screen, click the "Apple" icon. 
+   b. In the pop-up menu displayed, select `About this Mac`.
+   c. Information on your Mac system is displayed. 
```

Comment:
Use a hyphen `-` for bullet points here. I don't think we use the "a. b. c." method anywhere else in the docs.

nit: Change "pop-up menu" to "drop down menu".

Replace the backticks around "About this Mac" to quotation marks.

> c. Information on your Mac system is displayed.

"A window will appear displaying information about your Mac" or something like that.

Reply:
Changed the phrasing in https://github.com/ipfs/ipfs-docs/pull/1289/commits/8662022f8929748d6fd0d4f1001d07eb3fb08c55.

Question about the use of "a. b. c.": it seems like these should be ordered steps instead of unordered bullet points, since the user should do those steps in order, so I'm hesitant to use bullets. How have the ipfs docs handled stubsteps in other sections? I use "a. b. c." because that was a style I've used in the past. Open to whatever the current style is though.


Question about the use of backticks: I've used backticks in the past to denote both buttons and code. In this case, I was trying to indicate to the use that "About this Mac" is a button in the drop down. So, same question. How does the ipfs docs currently indicate a button? I'm open to different ways.

Reply:
For the steps, we use `1. 2. 3. etc.`
When writing the doc in markdown, I only use 1s. I believe it is a feature of Vuepress to take all those 1s and render them as the correct numbers in order. For example:
```
1. In the upper left hand corner of your screen, click the "Apple" icon. 
1. In the pop-up menu displayed, select `About this Mac`.
1. Information on your Mac system is displayed.
```

As for the second Q about backticks, I'm just re-reading the `Contributing` file now and it says, 
>"Things that the user must click on, or look out for, should be in bold.
>
>Click **Log out** to end your session. Enter your file name in the **File Name** field.

So, bit of a grey area there on whether to use backticks, quotation marks, or neither. Lets just follow the contributing guidelines for now and just make the buttons bold.